ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -2459,6 +2522,7 @@ namespace dpct
                 batch_size);
             break;
         }
+#endif
```

Comment:
Why are we reverting the type_combinations for non `__INTEL_MKL__` (namely Nvidia Targets) here ? - this was introduced in https://github.com/ggerganov/llama.cpp/pull/8236 and non Intel targets shall support these `gemm_batch` type-configurations.